src/encoded/docs/schema-changes.md

```diff
@@ -1,17 +1,17 @@
 Making changes to encoded schemas
 =========================
 
-This document describes how to make changes to the JSON schemas ([JSONSchema], [JSON-LD]) and source code that describes the encoded metadata model.  For overview of code organization see [overview.rst].
+This document describes how to make changes to the JSON schemas ([JSONSchema], [JSON-LD]) and source code that describes the encoded metadata model. For overview of code organization see [overview.rst].
 
 Guide to where to edit Source Code
 ----------------
 
 * **src** directory - contains all the python and javascript code for front and backends
-    * **audit** - python instructions for checking metadata stored in the schema
+    * **audit** - contains python scripts that check metadata stored in the schema and triggers audits in cases as defined here.
```

Comment:
* **audit** - contains python scripts that check json objects' metadata stored in the schema